webpack.common.js

```diff
@@ -50,11 +50,6 @@ function configure(IS_TEST, IS_INSTRUMENTED) {
           use: ['cache-loader', 'ng-annotate-loader', 'babel-loader', 'eslint-loader'],
           include: inclusionPattern.concat(path.resolve(__dirname, 'settings.js')),
         },
-        {
-          test: /\.js$/,
-          use: ['ify-loader', 'transform-loader?plotly.js/tasks/util/compress_attributes.js'],
```

Comment:
this is a plotly thing, they apparently do weird stuff with attributes and require a special build step to be commonjs-able